docs/agent-api.asciidoc

```diff
@@ -324,14 +324,82 @@ Whether or not the agent should monitor for uncaught exceptions and send them to
 
 Set this option to `false` to disable capture of stack traces for measured traces during instrumentation.
 
-[[source-context]]
-===== `sourceContext`
+[[source-context-error]]
+===== `sourceContextErrorAppFrames` + `sourceContextErrorLibraryFrames`
 
-* *Type:* Boolean
-* *Default:* `true`
-* *Env:* `ELASTIC_APM_SOURCE_CONTEXT`
+When an error is captured by the agent,
+its stack trace is stored in Elasticsearch.
+
+By default the agent will also collect a few lines of source code around the lines for each frame in the stack trace.
+This can make it easier to determine the cause of an error as the source code related to the error is visible directly in Kibana.
+
+The agent differentiates between so called in-app frames and library frames.
+Library frames are frames belonging to Node core and code inside the applications `node_modules` folder.
+In-app frames are everything else.
+
+Use the following 2 config options to change how many lines of source code to include for the different types of stack frames:
```

Comment:
I usually spell out small numbers in text paragraphs (http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Numbers.html?page=1)

Reply:
Good point

Reply:
Fixed 😃